### ISSUE TITLE
Add context for page numbers for screenreaders

### ISSUE THREAD

**Issue by StephDriver** [openlibhums/janeway#4247]
Page numbers need context for screen readers.  The following examples were identified in #4194 

- Sample 4, item 41. pages: 1-13 is read as "pages one thirteen" because the - is considered invisible punctuation as it could in say 'copy-editing` - this should read as "Pages one to thirteen" or similar. It should be described to be read that way.
- Sample 5.1, item 16. the page numbers at the end of the article details, '1-9' are read as 'one nine' because the screenreader doesn't know that punctuation is important in this place - suggest marking up both to describe these as pages, and to indicate it is a range, e.g. 'pages one to nine' - but need to be careful with implementing this as I believe the UK is 'one to nine' but the US preference may be for 'one through nine'.

**Comment by StephDriver** [comment]
To clarify: this issue is about use of page numbers within django templates.

When page numbers may also be used within the text of an article, but this would be either in paragraph text, or within a formatted reference, such that the context already shows it is a page number. The issue here is about a lack of context for the page numbers.  This would be when they are used in the django template only.

**Comment by StephDriver** [comment]
This could be resolved with ARIA, but in some cases it may be more accessible to provide context through text available to everyone (e.g. `Page 1-11` rather than `1-11` with some ARIA markup) .  Please consider each occurrence individually to determine whether the numbers are given enough visual context, as well as whether a screenreader user receives their context.

**Comment by StephDriver** [comment]
Update:
- Sample 4 issue resolved as there is now more context.
- Sample 5.1 issue is addressed in #4211 which makes the display titles screenreadable.

additional work was found around pagination, which is in the PR.
